<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="3" failed="7" ignored="0" total="27" passed="17">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="76056" started-at="2020-09-22T11:28:01 PDT" finished-at="2020-09-22T11:29:17 PDT">
    <groups>
    </groups>
    <test name="Test" duration-ms="76056" started-at="2020-09-22T11:28:01 PDT" finished-at="2020-09-22T11:29:17 PDT">
      <class name="PHPTravel.End2EndExample.LandingPageHotelsTest">
        <test-method status="PASS" signature="initialize()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHotelsTest@480bdb19]" name="initialize" is-config="true" duration-ms="14097" started-at="2020-09-22T11:28:16 PDT" finished-at="2020-09-22T11:28:30 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="FAIL" signature="checkinCalendarTitle()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHotelsTest@480bdb19]" name="checkinCalendarTitle" duration-ms="2281" started-at="2020-09-22T11:29:01 PDT" finished-at="2020-09-22T11:29:03 PDT">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [September,2020] but found [September,
2020]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [September,2020] but found [September,
2020]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at PHPTravel.End2EndExample.LandingPageHotelsTest.checkinCalendarTitle(LandingPageHotelsTest.java:121)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkinCalendarTitle -->
        <test-method status="PASS" signature="currentDayHighlightedCheckin()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHotelsTest@480bdb19]" name="currentDayHighlightedCheckin" duration-ms="127" started-at="2020-09-22T11:29:03 PDT" finished-at="2020-09-22T11:29:03 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- currentDayHighlightedCheckin -->
        <test-method status="PASS" signature="verifyDestinationDefaultText()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHotelsTest@480bdb19]" name="verifyDestinationDefaultText" duration-ms="92" started-at="2020-09-22T11:29:03 PDT" finished-at="2020-09-22T11:29:03 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyDestinationDefaultText -->
        <test-method status="PASS" signature="verifyDestinationSuggest()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHotelsTest@480bdb19]" name="verifyDestinationSuggest" duration-ms="6853" started-at="2020-09-22T11:29:03 PDT" finished-at="2020-09-22T11:29:10 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyDestinationSuggest -->
        <test-method status="FAIL" signature="verifyECheckinDefaultDate()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHotelsTest@480bdb19]" name="verifyECheckinDefaultDate" duration-ms="23" started-at="2020-09-22T11:29:10 PDT" finished-at="2020-09-22T11:29:10 PDT">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [22/09/2020] but found [25/09/2020]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [22/09/2020] but found [25/09/2020]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at PHPTravel.End2EndExample.LandingPageHotelsTest.verifyECheckinDefaultDate(LandingPageHotelsTest.java:97)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyECheckinDefaultDate -->
        <test-method status="PASS" signature="verifyOnClickCheckin()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHotelsTest@480bdb19]" name="verifyOnClickCheckin" duration-ms="235" started-at="2020-09-22T11:29:10 PDT" finished-at="2020-09-22T11:29:10 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyOnClickCheckin -->
        <test-method status="FAIL" signature="verifyTabText()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHotelsTest@480bdb19]" name="verifyTabText" duration-ms="127" started-at="2020-09-22T11:29:10 PDT" finished-at="2020-09-22T11:29:10 PDT">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [HOTELS] but found [Hotels]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [HOTELS] but found [Hotels]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at PHPTravel.End2EndExample.LandingPageHotelsTest.verifyTabText(LandingPageHotelsTest.java:44)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyTabText -->
        <test-method status="FAIL" signature="verifyTitlesText()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHotelsTest@480bdb19]" name="verifyTitlesText" duration-ms="216" started-at="2020-09-22T11:29:10 PDT" finished-at="2020-09-22T11:29:11 PDT">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Lists differ at element [3]: ADULTS (13-75) != ADULTS (12-75) expected [ADULTS (13-75)] but found [ADULTS (12-75)]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Lists differ at element [3]: ADULTS (13-75) != ADULTS (12-75) expected [ADULTS (13-75)] but found [ADULTS (12-75)]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:1030)
at org.testng.Assert.assertEquals(Assert.java:992)
at PHPTravel.End2EndExample.LandingPageHotelsTest.verifyTitlesText(LandingPageHotelsTest.java:65)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyTitlesText -->
        <test-method status="SKIP" signature="checkinForwardArrow()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHotelsTest@480bdb19]" name="checkinForwardArrow" duration-ms="0" started-at="2020-09-22T11:29:11 PDT" depends-on-methods="PHPTravel.End2EndExample.LandingPageHotelsTest.verifyECheckinDefaultDate, PHPTravel.End2EndExample.LandingPageHotelsTest.checkinCalendarTitle" finished-at="2020-09-22T11:29:11 PDT">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method LandingPageHotelsTest.checkinForwardArrow()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHotelsTest@480bdb19] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method LandingPageHotelsTest.checkinForwardArrow()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHotelsTest@480bdb19] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:102)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkinForwardArrow -->
        <test-method status="SKIP" signature="checkinBackArrow()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHotelsTest@480bdb19]" name="checkinBackArrow" duration-ms="0" started-at="2020-09-22T11:29:11 PDT" depends-on-methods="PHPTravel.End2EndExample.LandingPageHotelsTest.checkinForwardArrow, PHPTravel.End2EndExample.LandingPageHotelsTest.verifyECheckinDefaultDate, PHPTravel.End2EndExample.LandingPageHotelsTest.checkinCalendarTitle" finished-at="2020-09-22T11:29:11 PDT">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method LandingPageHotelsTest.checkinBackArrow()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHotelsTest@480bdb19] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method LandingPageHotelsTest.checkinBackArrow()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHotelsTest@480bdb19] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:102)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkinBackArrow -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHotelsTest@480bdb19]" name="teardown" is-config="true" duration-ms="5911" started-at="2020-09-22T11:29:11 PDT" finished-at="2020-09-22T11:29:17 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- PHPTravel.End2EndExample.LandingPageHotelsTest -->
      <class name="PHPTravel.End2EndExample.LandingPageHeaderTest">
        <test-method status="PASS" signature="initialize()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHeaderTest@86be70a]" name="initialize" is-config="true" duration-ms="15858" started-at="2020-09-22T11:28:01 PDT" finished-at="2020-09-22T11:28:16 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="PASS" signature="clickBackToHome()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHeaderTest@86be70a]" name="clickBackToHome" duration-ms="10606" started-at="2020-09-22T11:28:32 PDT" finished-at="2020-09-22T11:28:43 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickBackToHome -->
        <test-method status="PASS" signature="currencyDropDownVerification()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHeaderTest@86be70a]" name="currencyDropDownVerification" duration-ms="1840" started-at="2020-09-22T11:28:43 PDT" finished-at="2020-09-22T11:28:45 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- currencyDropDownVerification -->
        <test-method status="FAIL" signature="currencyTypes()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHeaderTest@86be70a]" name="currencyTypes" duration-ms="2288" started-at="2020-09-22T11:28:45 PDT" finished-at="2020-09-22T11:28:47 PDT">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:968)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at PHPTravel.End2EndExample.LandingPageHeaderTest.currencyTypes(LandingPageHeaderTest.java:114)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- currencyTypes -->
        <test-method status="PASS" signature="languageDefaultTextVerification()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHeaderTest@86be70a]" name="languageDefaultTextVerification" duration-ms="167" started-at="2020-09-22T11:28:47 PDT" finished-at="2020-09-22T11:28:47 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- languageDefaultTextVerification -->
        <test-method status="PASS" signature="myAccountDefaultText()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHeaderTest@86be70a]" name="myAccountDefaultText" duration-ms="196" started-at="2020-09-22T11:28:47 PDT" finished-at="2020-09-22T11:28:47 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- myAccountDefaultText -->
        <test-method status="FAIL" signature="callNowVerification()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHeaderTest@86be70a]" name="callNowVerification" duration-ms="210" started-at="2020-09-22T11:28:47 PDT" finished-at="2020-09-22T11:28:47 PDT">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Call Now: +1-360-244-9610] but found [phone Call Now : +1-234-56789]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Call Now: +1-360-244-9610] but found [phone Call Now : +1-234-56789]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at PHPTravel.End2EndExample.LandingPageHeaderTest.callNowVerification(LandingPageHeaderTest.java:78)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- callNowVerification -->
        <test-method status="PASS" signature="clickAboutUs()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHeaderTest@86be70a]" name="clickAboutUs" duration-ms="2041" started-at="2020-09-22T11:28:47 PDT" finished-at="2020-09-22T11:28:50 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickAboutUs -->
        <test-method status="PASS" signature="clickCompanyLogo()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHeaderTest@86be70a]" name="clickCompanyLogo" duration-ms="5033" started-at="2020-09-22T11:28:50 PDT" finished-at="2020-09-22T11:28:55 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickCompanyLogo -->
        <test-method status="PASS" signature="clickContact()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHeaderTest@86be70a]" name="clickContact" duration-ms="1540" started-at="2020-09-22T11:28:55 PDT" finished-at="2020-09-22T11:28:56 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickContact -->
        <test-method status="PASS" signature="currencyDefaultTextVerification()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHeaderTest@86be70a]" name="currencyDefaultTextVerification" duration-ms="157" started-at="2020-09-22T11:28:56 PDT" finished-at="2020-09-22T11:28:56 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- currencyDefaultTextVerification -->
        <test-method status="PASS" signature="languageDropDownVerification()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHeaderTest@86be70a]" name="languageDropDownVerification" duration-ms="600" started-at="2020-09-22T11:28:56 PDT" finished-at="2020-09-22T11:28:57 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- languageDropDownVerification -->
        <test-method status="FAIL" signature="languageTypes()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHeaderTest@86be70a]" name="languageTypes" duration-ms="661" started-at="2020-09-22T11:28:57 PDT" finished-at="2020-09-22T11:28:58 PDT">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:968)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at PHPTravel.End2EndExample.LandingPageHeaderTest.languageTypes(LandingPageHeaderTest.java:179)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- languageTypes -->
        <test-method status="PASS" signature="mouseOverCompany()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHeaderTest@86be70a]" name="mouseOverCompany" duration-ms="933" started-at="2020-09-22T11:28:58 PDT" finished-at="2020-09-22T11:28:59 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- mouseOverCompany -->
        <test-method status="PASS" signature="myAccountDropDownVerification()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHeaderTest@86be70a]" name="myAccountDropDownVerification" duration-ms="468" started-at="2020-09-22T11:28:59 PDT" finished-at="2020-09-22T11:28:59 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- myAccountDropDownVerification -->
        <test-method status="PASS" signature="selectLogin()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHeaderTest@86be70a]" name="selectLogin" duration-ms="917" started-at="2020-09-22T11:28:59 PDT" finished-at="2020-09-22T11:29:00 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- selectLogin -->
        <test-method status="SKIP" signature="currencySelection()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHeaderTest@86be70a]" name="currencySelection" duration-ms="0" started-at="2020-09-22T11:29:00 PDT" depends-on-methods="PHPTravel.End2EndExample.LandingPageHeaderTest.currencyTypes" finished-at="2020-09-22T11:29:00 PDT">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method LandingPageHeaderTest.currencySelection()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHeaderTest@86be70a] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method LandingPageHeaderTest.currencySelection()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHeaderTest@86be70a] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:102)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- currencySelection -->
        <test-method status="PASS" signature="languageSelection()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHeaderTest@86be70a]" name="languageSelection" duration-ms="540" started-at="2020-09-22T11:29:00 PDT" depends-on-methods="PHPTravel.End2EndExample.LandingPageHeaderTest.myAccountDefaultText" finished-at="2020-09-22T11:29:01 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- languageSelection -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:PHPTravel.End2EndExample.LandingPageHeaderTest@86be70a]" name="teardown" is-config="true" duration-ms="90" started-at="2020-09-22T11:29:11 PDT" finished-at="2020-09-22T11:29:11 PDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- PHPTravel.End2EndExample.LandingPageHeaderTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
